sickbeard/subtitles.py

```diff
@@ -849,6 +849,10 @@ def dhm(td):
                              ep_to_sub['show_name'], ep_num)
                 continue
 
+            if sickbeard.SUBTITLES_STOP_AT_FIRST and ep_to_sub['subtitles']:
```

Comment:
It was hard to understand, but now I get it.

If you want a show in 2 or more languages, you could post pone the pp until one of them are available. Then you can post process it and watch the show. No need to find more subs.

That's the use case. Uncommon, I think... but it's still an use case.